### ISSUE TITLE
multiSnapshotWithOptions names snapshot like first file in current folder

### ISSUE THREAD

**Issue by oldwin** [storybooks/storybook#3152]
### Issue details

I'm using multiSnapshotWithOptions to generate multiple snapshots

```js
initStoryshots({
  test: multiSnapshotWithOptions({}),
});
```

For example I have next stories in the folder

**folder/BBB.stories.js**
```
storiesOf('myStory', module)
  .add('BBB', () => (
```

**folder/CCC.stories.js**
```
storiesOf('myStory', module)
  .add('CCC', () => (
```

I expect storyshot will create snapshot file with name 'myStory' or two files for BBB and CCC. But it creates __snapshots__/**BBB**.stories.storyshot (just first file in the folder in alphabetical order).

If I will create file AAA.stories.js and update snapshots it will remove __snapshots__/**BBB**.stories.storyshot and create new wile __snapshots__/**AAA**.stories.storyshot

### Please specify which version of Storybook and optionally any affected addons that you're running

- storybook/addon-actions: 3.3.14
- storybook/addon-knobs: 3.3.14
- storybook/addon-storyshots: 3.3.14
- storybook/addons: 3.3.14
- storybook/react: 3.3.14

**Comment by igor-dv** [comment]
I am going to work on storyshots after the 3.4. One of the things is to make multisnapshot more configurable.
CC. #2517

**Comment by oldwin** [comment]
@igor-dv hi, any updates? Maybe I can help with something?

**Comment by igor-dv** [comment]
Still didn't manage to work on the refactoring, but you are welcome to help

**Comment by igor-dv** [comment]
Will be able to do with extending the `Stories2SnapsConverter`.

**Comment by Hypnosphi** [comment]
Released as [`4.0.0-alpha.10`](https://github.com/storybooks/storybook/releases/tag/v4.0.0-alpha.10)
